releasetasks/release_configs/dev_mozilla-esr45_firefox_rc_graph_2.yml

```diff
@@ -30,12 +30,14 @@ checksums_enabled: false
 push_to_candidates_enabled: false
 update_verify_enabled: false
 push_to_releases_enabled: true
-# rc graph 2 only needs final_verify_channels
+# channels aren't needed for rc graph 2
 channels:
```

Comment:
It's channels field that it needs - https://hg.mozilla.org/build/tools/file/tip/buildfarm/release/releasetasks_graph_gen.py#l80, not `release_channels`. Same for all ocurrences.